SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -285,7 +287,7 @@ bool USpatialClassInfoManager::ShouldTrackHandoverProperties() const
 
 	const UAbstractLBStrategy* Strategy = NetDriver->LoadBalanceStrategy;
 	check(Strategy != nullptr);
-	return Strategy->RequiresHandoverData() || Settings->bEnableHandover;
+	return Strategy->RequiresHandoverData() && USpatialStatics::IsHandoverEnabled(NetDriver);
```

Comment:
The default value for bEnableHandover changed from false to true here but IsHandoverEnabled will now disable handover entirely so I think this is ok

Reply:
Why do we have `RequiresHandoverData` and `bEnableHandover` ?